src/main/java/jenkins/scm/api/mixin/ChangeRequestSCMHead.java

```diff
@@ -47,6 +49,36 @@
     @NonNull
     String getId();
 
+    /**
+     * Returns the {@link ChangeRequestCheckoutStrategy} of this {@link ChangeRequestSCMHead}.
+     *
+     * @return the {@link ChangeRequestCheckoutStrategy}.
+     * @since TODO
+     */
+    @CheckForNull
```

Comment:
Shouldn't we keep this `@NonNull`?

Reply:
It looks like there's an error here. It's conflict https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fscm-api-plugin/detail/PR-77/10/pipeline#step-91-log-204.